en/backup-to-gcs-using-br.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Back up Data to GCS Using BR
+summary: Learn how to back up data to Google Cloud Storage (GCS) using BR.
+---
+
+# Back up Data to GCS Using BR
+
+This document describes how to back up the data of a TiDB cluster in Kubernetes to [Google Cloud Storage](https://cloud.google.com/storage/docs/) (GCS). "Backup" in this document refers to full backup (ad-hoc full backup and scheduled full backup). [BR](https://pingcap.com/docs/stable/br/backup-and-restore-tool/) is used to get the logic backup of the TiDB cluster, and then this backup data is sent to GCS.
+
+The backup method described in this document is implemented using Custom Resource Definition (CRD) in TiDB Operator v1.1 or later versions.
+
+## Ad-hoc full backup
+
+Ad-hoc full backup describes the backup by creating a `Backup` Custom Resource (CR) object. TiDB Operator performs the specific backup operation based on this `Backup` object. If an error occurs during the backup process, TiDB Operator does not retry, and you need to handle this error manually.
```

Comment:
`you need` may be change to `customer need` ?

Reply:
I think the current expression is OK, the users are reading this doc.